pkg/vm/emit/emit.go

```diff
@@ -91,20 +91,20 @@ func Syscall(w *io.BinWriter, api string) {
 }
 
 // Call emits a call Instruction with label to the given buffer.
-func Call(w *io.BinWriter, op opcode.Opcode, label int16) {
+func Call(w *io.BinWriter, op opcode.Opcode, label uint16) {
 	Jmp(w, op, label)
 }
 
 // Jmp emits a jump Instruction along with label to the given buffer.
-func Jmp(w *io.BinWriter, op opcode.Opcode, label int16) {
+func Jmp(w *io.BinWriter, op opcode.Opcode, label uint16) {
```

Comment:
I don't really like this change, because semantically `jmp` and `call` parameters are `int16`. At the same time, if we to interpret it as a label, then it seems to be fine. Maybe worth writing a comment in the function description.